torch/csrc/jit/jit_opt_bisect.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+#include <torch/csrc/WindowsTorchApiMacro.h>
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+// `TorchScript` offers a simple bisection utility function
```

Comment:
We probably also need to add a description to `OVERVIEW.md`: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/OVERVIEW.md#jit-logging